src/index.js

```diff
@@ -1,17 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import rootReducer from './reducers';
 import App from './App';
-import * as serviceWorker from './serviceWorker';
 
+const store = createStore(rootReducer);
 ReactDOM.render(
-  <React.StrictMode>
+  <Provider store={store}>
     <App />
-  </React.StrictMode>,
-  document.getElementById('root')
+  </Provider>,
+  document.getElementById('root'),
```

Comment:
we need to move the book object created in `reducer/book.js` in here.

![image](https://user-images.githubusercontent.com/31897434/89564874-91c8f300-d815-11ea-8773-3892a0d7aba9.png)